Dashboard/app/js/templates/application/application.handlebars

```diff
@@ -29,6 +29,7 @@
       <div class="buttons menuCentre">
         <ul>
           {{#if FLOW.dialogControl.showOK}} <li><a {{action "doOK" target="FLOW.dialogControl"}} class="ok smallBtn">{{t _ok}}</a></li>{{/if}}
+          {{#if FLOW.dialogControl.showOKDisabled}} <li><a class="ok smallBtn disabled">{{t _ok}}</a></li>{{/if}}
```

Comment:
I think in this case you may want to use the framework facility `{{bindAttr }}` that is for exactly this kind of use case.

see an example here 
https://github.com/akvo/akvo-flow/blob/529c2aee5f55c3d109b909c238639d0172f90392/Dashboard/app/js/templates/application/application.handlebars#L21

Reply:
ok Fixed, thanks a lot for your help on this one @muloem